src/components/atoms/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import Link from "next/link";
+import newbieslogo from "@/assets/small-newbies-logo.svg";
```

Comment:
zaciągnij sobie najnowsze zmiany z maina, tam logo jest w formacie png. Skorzystaj proszę z niego bo to svg to takie udawane svg ze zdjęciem z base64 :)